orbit/cmd/orbit/orbit.go

```diff
@@ -1324,6 +1324,7 @@ func getHostInfo(osqueryPath string, osqueryDBPath string) (*osqueryHostInfo, er
 	log.Debug().Str("query", systemQuery).Msg("running single query")
 	out, err := exec.Command(osqueryPath, args...).Output()
 	if err != nil {
+		log.Debug().Str("output", string(out)).Msg("getHostInfo via osquery")
```

Comment:
Unrelated to the PR, but this could be helpful in debugging (not sure it would contain anything, but just in case).